src/main/webapp/js/finscholar/viewfactory/viewfactory.js

```diff
@@ -39,9 +38,6 @@ const navbarViewFactory = (index) => {
     case 2:
       view = new ScholarshipListView();
       break;
-    case 3:
-      view = new ScholarshipPageView();
-      break;
```

Comment:
Why are you removing this and why are there 2 cases for ScholarshipListView?

Reply:
College list view is not implemented in this branch. I just left case 1 blank

Reply:
Ok cool. In the future, don't add implementation for things that don't exist. Leave the case blank with just the break; statement and a comment explaining that the view doesn't exist yet.